userspace/libsinsp/plugin.cpp

```diff
@@ -124,14 +124,14 @@ class sinsp_filter_check_plugin : public sinsp_filter_check
 		return new sinsp_filter_check_plugin(*this);
 	}
 
-	uint8_t* extract(sinsp_evt *evt, OUT uint32_t* len, bool sanitize_strings)
+	bool extract(sinsp_evt *evt, OUT vector<extract_value_t>& values, bool sanitize_strings = true)
```

Comment:
I don't think we should specify default argument here too!

Reply:
Yeah unfortunately this is a declaration.